### PR TITLE
changes made to telemetry

### PR DESCRIPTION
**Pull request by ManeeshaShibu**

### CODE REVIEW

File: library/ws-widget/collection/src/lib/_common/discussions/discussions.component.ts [hunk 1]
```diff
@@ -28,7 +29,22 @@ export class DiscussionsComponent implements OnInit {
     this.countArr =  this.count === 2 ? [1, 2] : [1, 2, 3]
   }
 
-  handleSelectedDiscuss(discussData: any, trend: boolean): void {
-    this.router.navigateByUrl(`/app/discussion-forum/topic/${ trend ? discussData.slug : discussData.topic.slug }?page=home`)
-  }
-}
+  handleSelectedDiscuss(discussData: any, context: string | boolean): void {
+    console.log(context)
```
Comment (vishnubansaltarento) [inline library/ws-widget/collection/src/lib/_common/discussions/discussions.component.ts]: Please remove console

Comment (ManeeshaShibu) [reply library/ws-widget/collection/src/lib/_common/discussions/discussions.component.ts]: Ok

File: library/ws-widget/collection/src/lib/_common/discussions/discussions.component.ts [hunk 2]
```diff
@@ -28,7 +29,22 @@ export class DiscussionsComponent implements OnInit {
...
+    const subType = context && context === 'my-discussions' ? 'my-discussions' : 'trending-discussions';
+  
+    this.router.navigateByUrl(`/app/discussion-forum/topic/${discussData.slug}?page=home`);
+  
+    this.eventService.raiseInteractTelemetry(
```
Comment (vishnubansaltarento) [inline library/ws-widget/collection/src/lib/_common/discussions/discussions.component.ts:35]: Please confirm, if this piece of code come after routing

File: library/ws-widget/collection/src/lib/content-strip-with-tabs/content-strip-with-tabs.component.html [hunk 3]
```diff
@@ -86,7 +86,7 @@
                       <div *ngIf="stripsResultDataMap && stripsResultDataMap[key]?.showOnLoader ||
                       !tab.fetchTabStatus"
                       class="text-center my-2">
-                            <ws-utils-horizontal-scroller-v2 [sliderConfig]="stripsResultDataMap[key]?.sliderConfig"
+                            <ws-utils-horizontal-scroller-v2  *ngIf="stripsResultDataMap && key && stripsResultDataMap[key]"[sliderConfig]="stripsResultDataMap[key]?.sliderConfig"
```
Comment (vishnubansaltarento) [inline library/ws-widget/collection/src/lib/content-strip-with-tabs/content-strip-with-tabs.component.html:89]: Why this condition has been added

File: library/ws-widget/collection/src/lib/content-strip-with-tabs/content-strip-with-tabs.component.html [hunk 4]
```diff
@@ -110,35 +110,35 @@
...
+                      
                     </div>
                   </mat-tab>
                 </mat-tab-group>
               </ng-container>
               <ng-container *ngIf="!stripsResultDataMap[key].tabs || !stripsResultDataMap[key].tabs.length">
                 <div *ngIf="stripsResultDataMap && stripsResultDataMap[key]?.showOnLoader" class="text-center my-2">
-                      <ws-utils-horizontal-scroller-v2 [sliderConfig]="stripsResultDataMap[key]?.sliderConfig"
-                              [loadStatus]="'fetching'" [stripConfig]="stripsResultDataMap[key]?.stripConfig">
+                      <ws-utils-horizontal-scroller-v2 *ngIf="stripsResultDataMap && key && stripsResultDataMap[key]"[sliderConfig]="stripsResultDataMap[key]?.sliderConfig"
```
Comment (vishnubansaltarento) [inline library/ws-widget/collection/src/lib/content-strip-with-tabs/content-strip-with-tabs.component.html:131]: Same as above

File: library/ws-widget/collection/src/lib/content-strip-with-tabs/content-strip-with-tabs.component.html [hunk 5]
```diff
@@ -110,35 +110,35 @@
...
+                              [loadStatus]="'fetching'" [subtype]="stripsResultDataMap[key]?.stripTitle">
```
Comment (vishnubansaltarento) [inline library/ws-widget/collection/src/lib/content-strip-with-tabs/content-strip-with-tabs.component.html:132]: Please confirm this change

File: library/ws-widget/collection/src/lib/content-strip-with-tabs/content-strip-with-tabs.component.ts [hunk 6]
```diff
@@ -1180,4 +1188,9 @@ export class ContentStripWithTabsComponent extends WidgetBaseComponent
     return this.langtranslations.translateLabel(label, type, '')
   }
 
+  setActiveTabData(tabData: any) {
+    console.log('Active Tab:', this.activeTabData)
```
Comment (vishnubansaltarento) [inline library/ws-widget/collection/src/lib/content-strip-with-tabs/content-strip-with-tabs.component.ts]: Remove console

File: library/ws-widget/collection/src/lib/content-strip-with-tabs/content-strip-with-tabs.component.html (hunk 5 above)
Comment (ManeeshaShibu) [reply library/ws-widget/collection/src/lib/content-strip-with-tabs/content-strip-with-tabs.component.html:132]: yeah this is added to get the subtype

File: library/ws-widget/collection/src/lib/content-strip-with-tabs/content-strip-with-tabs.component.ts (hunk 6 above)
Comment (ManeeshaShibu) [reply library/ws-widget/collection/src/lib/content-strip-with-tabs/content-strip-with-tabs.component.ts]: ok

File: library/ws-widget/collection/src/lib/content-strip-with-tabs/content-strip-with-tabs.component.html (hunk 3 above)
Comment (ManeeshaShibu) [reply library/ws-widget/collection/src/lib/content-strip-with-tabs/content-strip-with-tabs.component.html:89]: It is to check whether that condition is true or not

File: library/ws-widget/collection/src/lib/_common/discussions/discussions.component.ts (hunk 2 above)
Comment (ManeeshaShibu) [reply library/ws-widget/collection/src/lib/_common/discussions/discussions.component.ts:35]: yes this is working

**Pull request merged**